airflow/__init__.py

```diff
@@ -31,7 +31,7 @@
 from airflow.models import DAG
 from flask_admin import BaseView
 from importlib import import_module
-from airflow.exceptions import AirflowException
+from airflow.exceptions import AirflowException, AirflowSkipException
```

Comment:
I don't think we need to import this at the root level because it's scope is relatively narrow.